packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -259,20 +261,23 @@ export default class ListControl extends React.Component {
   };
 
   renderListControl() {
+    console.log( this.props );
     const { value, forID, field, classNameWrapper } = this.props;
     const { itemsCollapsed } = this.state;
     const items = value || List();
```

Comment:
We need this value to be a list of viable blank items when a minimum is set. Not certain if a simple list of empty Maps would do the trick. Also need to confirm how this impacts list widgets that use `field` instead of `field`, and may not have objects as values.